openlibrary/solr/update_work.py

```diff
@@ -603,10 +603,9 @@ def add_list(name, values):
             add_list('publish_year', pub_years)
             add('first_publish_year', min(int(y) for y in pub_years))
 
-        if get_solr_next():
-            number_of_pages_median = pick_number_of_pages_median(editions)
-            if number_of_pages_median:
-                add('number_of_pages_median', number_of_pages_median)
+        number_of_pages_median = pick_number_of_pages_median(editions)
+        if number_of_pages_median:
```

Comment:
```suggestion
        if number_of_pages_median := pick_number_of_pages_median(editions):
```

Reply:
Oh, we can't use walrus in our code yet because it breaks cython (I've really got automate that so that it breaks CI!)